src/kind.ts

```diff
@@ -6,168 +6,215 @@ import * as artifact from '@actions/artifact';
 import * as glob from '@actions/glob';
 import process from 'process';
 
-const VersionInput: string = "version";
-const ConfigInput: string = "config";
-const ImageInput: string = "image";
-const NameInput: string = "name";
-const WaitInput: string = "wait";
-const KubeConfigInput: string = "kubeconfig";
-const SkipClusterCreationInput: string = "skipClusterCreation";
-const SkipClusterDeletionInput: string = "skipClusterDeletion";
-const SkipClusterLogsExportInput: string = "skipClusterLogsExport";
-const toolName: string = "kind";
+const VersionInput = 'version';
+const VerbosityInput = 'verbosity';
+const QuietInput = 'quiet';
+const ConfigInput = 'config';
+const ImageInput = 'image';
+const NameInput = 'name';
+const WaitInput = 'wait';
+const KubeConfigInput = 'kubeconfig';
+const SkipClusterCreationInput = 'skipClusterCreation';
+const SkipClusterDeletionInput = 'skipClusterDeletion';
+const SkipClusterLogsExportInput = 'skipClusterLogsExport';
+const toolName = 'kind';
 
 export class KindConfig {
-    version: string
-    configFile: string;
-    image: string;
-    name: string;
-    waitDuration: string;
-    kubeConfigFile: string;
-    skipClusterCreation: boolean
-    skipClusterDeletion: boolean
-    skipClusterLogsExport: boolean
-    constructor(version: string, configFile: string, image: string, name: string, waitDuration: string,
-        kubeConfigFile: string, skipClusterCreation: string, skipClusterDeletion: string, skipClusterLogsExport: string) {
-        this.version = version;
-        this.configFile = configFile;
-        this.image = image;
-        this.name = name;
-        this.waitDuration = waitDuration;
-        this.kubeConfigFile = kubeConfigFile;
-        this.skipClusterCreation = (skipClusterCreation == 'true');
-        this.skipClusterDeletion = (skipClusterDeletion == 'true');
-        this.skipClusterLogsExport = (skipClusterLogsExport == 'true');
-    }
+  verbosity: number;
+  quiet: boolean;
```

Comment:
Could you please add the two inputs at the end, so the diff is clear?

Reply:
Done